index.js

```diff
@@ -90,6 +93,11 @@ function stripLoc (doc, removeLocAtThisLevel) {
     delete doc.loc;
   }
 
+  if (doc.loc) {
```

Comment:
this could use a comment - im not fully certain why we'd want to delete these properties. linking to the issue should be sufficient